src/autogluon_dashboard/scripts/widgets/widget.py

```diff
@@ -0,0 +1,11 @@
+class Widget:
+    """
+    This parent class is used to define any Panel widgets that are used on the dashboard website.
+    These widgets can be independent or attached to a dashboard to control the plot.
+    """
+
+    def __init__(self, **kwargs):
+        self.__dict__.update(kwargs)
+
+    def create_widget(self):
```

Comment:
```python
@abstractmethod
def create_widget(self):
  raise NotImplementedError
```
Help to avoid people accidentally creating a Widget object and tries to create it.

Reply:
Done!